src/celengine/body.cpp

```diff
@@ -1178,10 +1178,7 @@ void Body::recomputeCullingRadius()
         r = max(r, rings->outerRadius);
 
     if (atmosphere)
-    {
-        r = max(r, atmosphere->height);
-        r = max(r, atmosphere->cloudHeight);
-    }
+        r += max(atmosphere->height, atmosphere->cloudHeight);
```

Comment:
this looks reasonable (if atmosphere stores height above surface) but then you should move this to line 1177 (before `if (rings)` otherwise you add atmosphere height to outer rings radius.

could you also provide actual values of getBoundingRadius(), atmosphere->height and atmosphere->cloudHeight for some bodies?

Reply:
for the earth, getBoundingRadius() is 6310.xxxx, atmosphere->height is 100, cloudHeight is 7.

Reply:
okay

Reply:
moved to before the ring check